### PR TITLE
nix: fix cli derivation

### PR DESCRIPTION
**Pull request by tomhoule**
udf wrapper needs to be packaged

### CODE REVIEW

File: cli/udf-wrapper/package-lock.json
Comment (tomhoule) [inline cli/udf-wrapper/package-lock.json:1]: switch from package-lock v3 to v2 (tooling hasn't caught up to v3 yet)

File: cli/nix/udf-wrapper.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ pkgs, ... }:
+
+{
+  packages.udf-wrapper = pkgs.buildNpmPackage {
```
Comment (Finistere) [inline cli/nix/udf-wrapper.nix:4]: Does it use the build script defined in `package.json`?

Comment (tomhoule) [reply cli/nix/udf-wrapper.nix:4]: I think it does, yes

**Review by Finistere — APPROVED** [review]
Nice work!

**Pull request merged**